airbyte-webapp/src/components/StatusIcon/CircleLoader.module.scss

```diff
@@ -0,0 +1,25 @@
+@use "../../scss/colors";
+
+@keyframes spinning {
+  0% {
+    transform: rotate(0deg);
+  }
+
+  100% {
+    transform: rotate(360deg);
+  }
+}
+
+.spinner {
+  animation: spinning 1000ms linear infinite normal forwards;
+}
+
+.centerDot {
+  fill: colors.$blue;
+}
+
+:export {
+  // Export colors to be used in SVG gradients
+  gradientStart: colors.$blue;
```

Comment:
:information_source: `linear-gradient` doesn't work for `fill` on SVG elements, thus the gradient still need to be defined in SVG. To get access to the colors in JS, I exported them from here explicitally. They are already exported from the `theme.ts` file, but I consider this still to be going away once we're removing styled components, so in this case where we'll need to have this export even with SASS, I wanted to make it more explicit.